Loop/Extensions/UIImage.swift

```diff
@@ -39,3 +39,13 @@ extension UIImage {
         return UIImage(named: selected ? "workout-selected" : "workout")
     }
 }
+
+private class FrameworkBundle {
+    static let main = Bundle(for: FrameworkBundle.self)
```

Comment:
@ps2 had raised concerns with propagating this software pattern.  Just raising awareness, not blocking.